api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -436,6 +436,10 @@ type FelixConfigurationSpec struct {
 	// (ie it uses the iptables MASQUERADE target)
 	NATOutgoingAddress string `json:"natOutgoingAddress,omitempty"`
 
+	// When set to true and ip pool setting `natOutgoing` is true, packets sent from Calico networked containers in this pool
+	// to cluster host subnet will not be excluded from being masqueraded.  [Default: false]
+	DisableHostSubnetNATExclusion bool `json:"disableHostSubnetNATExclusion,omitempty"`
+
```

Comment:
Too many negatives for me to make much sense out of this. Could you make it a bit easier to read? Perhaps rename it to just `HostSubnetSNATExclusion`. And expand in the PR description on the two behaviours?

Reply:
I have made the adjustments. PTAL!